CONTRIBUTING.md

```diff
@@ -4,6 +4,11 @@
 
 There are a few basic ground-rules for contributors to `s`:
 
-1. When adding a `provider` use for each a new **single** commit.
-2. Use gofmt on all files.
-3. Refactor commits to the base code should be discussed in an issue prior to PR. 
\ No newline at end of file
+1. When adding a `provider`: 
+    1. Use for each a new **single** commit.
+    2. Add it to the README.md
+    3. Add it to `s.go`
+    4. Keep it all alphabetically
+    5. Create a PR.
+2. When doing refactor changes to the base code, first this should be discussed in an issue prior to PR. 
```

Comment:
"When doing refactors to the base code, make sure to discuss it in an issue prior to PR."